app/controllers/spotlight/catalog_controller.rb

```diff
@@ -197,7 +197,8 @@ def uploaded_resource_params
     end
 
     def custom_field_params
-      current_exhibit.custom_fields.writeable.pluck(:field)
+      current_exhibit.custom_fields.writeable.pluck(:slug) +
```

Comment:
Using the old name in strong params may be unnecessary, but done out of an abundance of caution. 🤷‍♂